app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
+class YourModelAdmin(admin.ModelAdmin):
+    list_display = ('field1', 'field2', 'field3')  # Add fields you want to display in the admin list view
```

Comment:
**Suggestion**: Consider adding more fields to the 'list_display' attribute for a comprehensive admin display. <br> **Code Suggestion**: 
 ```
 +    list_display = ('field1', 'field2', 'field3')
 ```